src/main/assets/scss/adoption/components/alert/_alert.scss

```diff
@@ -0,0 +1,13 @@
+.alert {
```

Comment:
What's our strategy with scss? Are we pulling it from Eilis prototype? Or it's something we need to code on our end?

Reply:
@akomisarek Eilis will be using the gov uk frontend scss which we are pulling into our project. But she's also used some experimental elements that are not yet included as a component (alert for example). 

We should however treat the prototype as a 'copy and paste' job and should always write the HTML ourselves.

Reply:
thanks for explanation, that's area which is a bit alien for me :P